Sources/SuperValidator/Validators/Phone.swift

```diff
@@ -56,14 +56,18 @@ extension SuperValidator {
                 return .failure(PhoneError.invalidPhone)
             }
             
-            let phoneCode = string.prefix(3)
             if options.specifiedCountryCode.isNotEmpty {
+                var phoneCode = ""
+                if string.first == "0" {
+                    phoneCode = "62"
```

Comment:
Hmm do we need to hardcode the country code to `62` when the prefix number is zero? IMHO regardless specifiedCountryCode that the user provides, we can treat it as a valid phone number
not 100% sure sih,  let's ask suggestions from others ya

Reply:
yeah hardcoded to 62 might be not best practice, but is there any solution to figure out, when the user wanna specify the country code (ex: 62) while our phone number start with 0 kakak Ambuy? Thank you

Reply:
hmm please do your research ted.
do you sure only Indonesia number start with 0 ?

Reply:
already fixed